tests/e2e/README.md

```diff
@@ -9,6 +9,16 @@ Developers, on the other hand, are recommended to run the tests locally before s
 
 # Running E2E Tests
 
+Image registry option:
+
+1. [In-cluster registry](/local/localregistry) 
+If environment variable `HUB` is not set, local registry will be setup and used automatically.
```

Comment:
Maybe expand this just a little:

If the environment variables `HUB` and `TAG` are *not* set, the test setup will automatically deploy a docker registry to the cluster.  New docker images will be built and pushed to this in-cluster registry and all test deployments will pull their images from this registry.

WDYT?